docs/developer-guide/maps-configuration.md

```diff
@@ -31,7 +31,9 @@ This is used for the **new map**. If you're logged in and allowed to create maps
 http://localhost:8081/#viewer/openlayers/new
 ```
 
-This is a special context that uses the `new.json` file in the root of the project. (`web/client` for standard mapstore, root for custom projects). You can edit `new.json` to customize the initial template for new maps (for instance, you can change the backgrounds).
+This page uses the `new.json` file as a template configuration to start creating a new map. You can find this file in `web/client` directory for standard MapStore or in the root for a custom projects. 
```

Comment:
Have a look here for another option (using the datadir): https://mapstore.readthedocs.io/en/latest/developer-guide/externalized-configuration/
It's a recent addition

Reply:
updated